query.go

```diff
@@ -2090,6 +2090,69 @@ func (q *Query) BufferSizeVar(attributeOrDimension string) (uint64, uint64, erro
 	return offsetNumElements, dataNumElements, nil
 }
 
+// BufferSizeVarNullable returns the size (in num elements) of the backing C buffers for the given variable-length nullable attribute
+func (q *Query) BufferSizeVarNullable(attributeName string) (uint64, uint64, uint64, error) {
+	var datatype Datatype
+	schema, err := q.array.Schema()
+	if err != nil {
+		return 0, 0, 0, err
+	}
+
+	domain, err := schema.Domain()
```

Comment:
The below code is also wrong because of this. You don't fetch the datatype of the attribute but actually looks like it unset except for TILEDB_COORS? That is an impossible condition cause TILEDB_COORDS can't be nullable. This should be handling and checking attributes.